tests/autotune_test_yamls/manifests/da/app_autotune_yaml/datasource_url/blank-datasource-url.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: "recommender.com/v1"
+kind: "Autotune"
+metadata:
+ name: "blank-datasource-url"
+spec:
+  sla:
+    objective_function: "transaction_response_time"
+    sla_class: "response_time"
+    direction: "minimize"
+    function_variables:
+    - name: "transaction_response_time"
+      query: "application_org_acme_microprofile_metrics_PrimeNumberChecker_checksTimer_mean_seconds"
+      datasource: "prometheus"
+      value_type: "double"
+  mode: "show"
+  selector:
+    matchLabel: "app.kubernetes.io/name"
+    matchLabelValue: "petclinic-deployment"
+    matchRoute: ""
+    matchURI: ""
+    matchService: "https"
+  datasource:
+    name: "prometheus"
```

Comment:
You should not be using "prometheus" for all datasource  tests, change it to prometheus1 or something. Also add a test with multiple datasources.